src/TwigComponent/doc/index.rst

```diff
@@ -1634,7 +1634,7 @@ Anonymous Components
 
 .. versionadded:: 2.20
 
-    The bundle convention for Anonymous components was added in TwigComponents 2.18.
+    The bundle convention for Anonymous components was added in TwigComponents 2.20.
```

Comment:
I'm jumping on the occasion... what do you think about something like "The automatic registration of Anonymouns components from Bundle" ? 

Just a suggestion, feel free to ignore if you prefer the current one :)

Reply:
Simon just saw it, feel free to improve it in a new PR, LGTM 👍